script/upgrades/MU08/MU08.sol

```diff
@@ -0,0 +1,251 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// solhint-disable func-name-mixedcase, contract-name-camelcase, function-max-lines, var-name-mixedcase
+pragma solidity ^0.5.13;
+pragma experimental ABIEncoderV2;
+
+import { GovernanceScript } from "script/utils/Script.sol";
+import { console } from "forge-std/console.sol";
+import { Contracts } from "script/utils/Contracts.sol";
+import { Chain } from "script/utils/Chain.sol";
+import { Arrays } from "script/utils/Arrays.sol";
+
+import { IMentoUpgrade, ICeloGovernance } from "script/interfaces/IMentoUpgrade.sol";
+import { IGovernanceFactory } from "script/interfaces/IGovernanceFactory.sol";
+
+interface IOwnableLite {
+  function owner() external view returns (address);
+
+  function transferOwnership(address recipient) external;
+}
+
+interface IProxyLite {
+  function _getImplementation() external view returns (address);
+}
+
+contract MU08 is IMentoUpgrade, GovernanceScript {
+  using Contracts for Contracts.Cache;
+
+  bool public hasChecks = true;
+  ICeloGovernance.Transaction[] private transactions;
+
+  // Celo Governance:
+  address private celoGovernance;
+
+  // Mento contracts:
+
+  //Tokens:
+  address private cUSDProxy;
+  address private cEURProxy;
+  address private cBRLProxy;
+  address private eXOFProxy;
+  address private cKESProxy;
+  //address private POSProxy;
```

Comment:
let's removed all the commented out code related to POSP?

Reply:
thought of it as a reminder to add it once PUSO is live